cmds/core/kexec/kexec_linux.go

```diff
@@ -16,7 +16,7 @@
 //  -c, --cmdline string       Append to the kernel command line
 //  -d, --debug                Print debug info (default true)
 //  -e, --exec                 Execute a currently loaded kernel
-//  -x, --extra string         Add a cpio containing extra files
```

Comment:
No change requested in this PR, but just get your view on this, so we guide review for future PRs:

How do you like about we removing this flags comments ?  we have flags defeinition down below, and I have seen following problems 

(1) New flags added in code are not commented in comments.
(2) Updated flag description are not always synced into comments.

Reply:
I agree. the flags comments are from an earlier time and I think they are not a good idea.